index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <link rel="stylesheet" href="main.css">
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+</head>
+
+<body>
+ <header>
+  <div class="header-left">
+   <h2>CodePen Home
+    FCC: Tribute Page</h2>
+  </div>
+
+  <div class="header-right">
+   <ul>
+    <li><a>Love</a></li>
+    <li><a>Change View</a></li>
+    <li><a>Sign Up</a></li>
+    <li><a>Log in</a></li>
+   </ul>
+  </div>
+ </header>
+ <div id="main">
```

Comment:
I think you should use semantic tag <main> instead of div

Reply:
It is really great that you noticed that not enough semantic tags were used. However, you should explain the reason why you are asking for the usage of semantic tags. This is a teachable moment for the student and he/she will have the opportunity to learn something.

Reply:
I think that you should use semantic tag <main> instead of a div with a class of main.
Semantic tags gives your markup more meaning to the web browser and screen readers,they also can help with the search engines.In the future you will work with a team and semantic tags will help you and your partners to understand the code faster, and its easier to find bugs.